dev/TabView/TabView.idl

```diff
@@ -178,7 +178,10 @@ unsealed runtimeclass TabViewItemTemplateSettings : Windows.UI.Xaml.DependencyOb
 
     Windows.UI.Xaml.Controls.IconElement IconElement;
 
+    Windows.UI.Xaml.Media.Geometry TabGeometry;
```

Comment:
This needs to use version v2 tag to keep ABI compat.

Reply:
Updated!